spaceship/spec/connect_api/fixtures/testflight/beta_testers.json

```diff
@@ -7,7 +7,27 @@
         "lastName" : "AreCute",
         "email" : "email@email.com",
         "inviteType" : "EMAIL",
-        "invitation" : null
+        "isDeleted": false,
+        "betaTesterState": "INSTALLED",
+        "lastModifiedDate": "2024-01-21T20:52:18.921-08:00",
+        "installedCfBundleShortVersionString": "1.3.300",
+        "installedCfBundleVersion": "1113",
+        "removeAfterDate": "2024-04-20T00:00:00-07:00",
+        "latestExpiringCfBundleShortVersionString": "1.3.300",
```

Comment:
```
       "latestExpiringCfBundleShortVersionString": "1.3.300",
        "latestExpiringCfBundleVersionString": "1113",
        "installedDevicePlatform": "IOS",
        "latestInstalledDevice": "iPhone14_7",
        "latestInstalledOsVersion": "17.2.1",
        "latestInstalledDevicePlatform": "IOS",
```

do we want to add these?

Reply:
Let's add them and update the test. OK for you @mlch911 ?

Reply:
No problem.